fpm/src/fpm.f90

```diff
@@ -174,4 +184,20 @@ subroutine cmd_build()
 call run("gfortran main.o " // linking // " -o " // pkg_name)
 end subroutine
 
+subroutine cmd_install()
+    print *, "fpm error: 'fpm install' not implemented."
```

Comment:
Put `error stop` underneath here?

Reply:
Good call, I forgot about it.

Reply:
Done. I also added non-zero exit code to all `error stop` statements. My gfortran was not doing that by default.